pkgs/development/python-modules/fitter/default.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  click,
+  joblib,
+  loguru,
+  matplotlib,
+  numpy,
+  pandas,
+  pytestCheckHook,
+  pytest-cov,
+  pytest-mock,
+  pytest-timeout,
+  pytest-xdist,
+  rich-click,
+  scipy,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "fitter";
+  version = "1.7.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "cokelaer";
+    repo = "fitter";
+    rev = "v${version}";
```

Helps to avoid branch/tag confusion.

```suggestion
    rev = "refs/tags/v${version}";
```